internal/exectracetest/go.mod

```diff
@@ -64,16 +64,16 @@ require (
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.27.0 // indirect
 	golang.org/x/mod v0.20.0 // indirect
-	golang.org/x/net v0.28.0 // indirect
+	golang.org/x/net v0.30.0 // indirect
 	golang.org/x/sync v0.8.0 // indirect
-	golang.org/x/sys v0.24.0 // indirect
-	golang.org/x/text v0.17.0 // indirect
+	golang.org/x/sys v0.26.0 // indirect
+	golang.org/x/text v0.19.0 // indirect
 	golang.org/x/time v0.6.0 // indirect
 	golang.org/x/tools v0.24.0 // indirect
 	golang.org/x/xerrors v0.0.0-20231012003039-104605ab7028 // indirect
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20240730163845-b1a4ccb954bf // indirect
-	google.golang.org/grpc v1.64.1 // indirect
-	google.golang.org/protobuf v1.34.2 // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20241021214115-324edc3d5d38 // indirect
+	google.golang.org/grpc v1.67.1 // indirect
```

Comment:
@rarguelloF Is this upgrade (and related) needed? If yes, we'll have to clearly state we support this integration from a specific version.

Reply:
`encoding.RegisterCodecV2` was not available on the previous version (in the one we were using, there was `encoding.RegisterCodec`). I am unsure what happens in someone uses the latest grpc, which registers the default codec using the `V2` function, and we register ours using the `V1`.

> we'll have to clearly state we support this integration from a specific version.
I think this is implicit from the `go.mod` right? since user's grpc version will automatically upgrade when they use this dd-trace-go version.

Reply:
We could add a comment just to make sure we also know from which version is available. WDYT?